spec/controllers/admin/rooms_controller_spec.rb

```diff
@@ -0,0 +1,188 @@
+require 'spec_helper'
+
+describe Admin::RoomsController do
+  let!(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let!(:venue) { create(:venue, conference: conference) }
+  let(:room) { create(:room, venue: venue, size: 4) }
+
+  context 'non admin user is signed in' do
```

Comment:
It's still wrong :-) Please test `BaseController#verify_user_admin` instead of all the descendants.